src/util/psbt/map/input.rs

```diff
@@ -12,18 +12,19 @@
 // If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
 //
 
-use std::collections::BTreeMap;
+use std::collections::{btree_map::Entry, BTreeMap};
```

Comment:
nit: this multi-leveled use statements are quite confusing, I don't think we use them anywhere in this crate yet.

I'd prefer 
```rust
use std::collections::btree_map::{Entry, BTreeMap};
```